src/components/TodoList.js

```diff
@@ -163,6 +164,19 @@ const mapDispatchToProps = (dispatch, ownProps) => {
           todoListId: ownProps.id
         }
       })
+    },
+
+    snoozeTodo: (todo, numberOfDays) => {
+      let newTodoDate = moment(todo.dueDate).add(numberOfDays, 'days')
```

Comment:
Use `const` instead of `let`, as you don't need to modify the value referenced by `newTodoDate`. In general, you will very rarely use `let`.
I found [this article](https://putaindecode.io/articles/es6-es2015-la-declaration-de-variables-avec-const-let-et-var/) talking about `const`, `let` and `var`. It might be interesting.
It is important that you understand the language behind React and Redux, which are frameworks **built on javascript**. It will be important to understand it well as you plan on specializing on javascript for now.

Concerning `newTodoDate = moment(todo.dueDate).add(numberOfDays, 'days')`, well done ! :)